.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+
+Thank you for the PR! Contributors like you keep React awesome!
+
+Please see the Contribution Guide for guidelines:
+
+https://github.com/reactjs/reactjs.org/blob/main/CONTRIBUTING.md
+
+If your PR references an existing issue, please add the issue number below
+
+-->
+
+### Problem
+
+<!--
+What problem are you solving with this PR?
+-->
+
+(Write problem here.)
+
+### Solution
+
+<!-- What solution did you implement? -->
+
+### How did you test this change?
```

Comment:
This template seems a bit heavy-handed for e.g. typos

Reply:
Shall we keep it basic for typos and add separate section? would that help ?